The *-grpc targets were recently removed in google-cloud-cpp

https://github.com/googleapis/google-cloud-cpp/pull/12939/files/300968f8921e4d978a712570203abc0f717cb42e

https://github.com/googleapis/google-cloud-cpp/issues/12698

--- a/ports/google-cloud-cpp/vcpkg.json
+++ b/ports/google-cloud-cpp/vcpkg.json
@@ -475,7 +475,7 @@
         }
       ]
     },
-    "experimental-storage-grpc": {
+    "experimental-storage_grpc": {
       "description": "The GCS+gRPC plugin",
       "dependencies": [
         {
--- a/ports/google-cloud-cpp/portfile.cmake
+++ b/ports/google-cloud-cpp/portfile.cmake
@@ -49,7 +49,7 @@ foreach(feature IN LISTS FEATURES)
     set(config_path "lib/cmake/google_cloud_cpp_${feature}")
     # Most features get their own package in `google-cloud-cpp`.
     # The exceptions are captured by this `if()` command, basically
-    # things like `core` and `experimental-storage-grpc` are skipped.
+    # things like `core` and `experimental-storage_grpc` are skipped.
     if(NOT IS_DIRECTORY "${CURRENT_PACKAGES_DIR}/${config_path}")
         continue()
     endif()
--- a/cmake/Modules/FindGCSSDK_EP.cmake
+++ b/cmake/Modules/FindGCSSDK_EP.cmake
@@ -177,6 +177,6 @@ if (GCSSDK_FOUND AND NOT TARGET storage_
     endif()
   elseif(google_cloud_cpp_storage_FOUND)
     add_library(storage_client INTERFACE IMPORTED)
-    target_link_libraries(storage_client INTERFACE CURL::libcurl google-cloud-cpp::storage google-cloud-cpp::experimental-storage-grpc)
+    target_link_libraries(storage_client INTERFACE CURL::libcurl google-cloud-cpp::storage google-cloud-cpp::experimental-storage_grpc)
   endif()
 endif()
--- a/tiledb/CMakeLists.txt
+++ b/tiledb/CMakeLists.txt
@@ -1014,7 +1014,7 @@ if (TILEDB_STATIC)
   append_dep_lib(LibLZMA::LibLZMA)
 
   append_dep_lib(google-cloud-cpp::storage)
-  append_dep_lib(google-cloud-cpp::experimental-storage-grpc)
+  append_dep_lib(google-cloud-cpp::experimental-storage_grpc)
 
   append_dep_lib(libmagic)
   append_dep_lib(unofficial::libmagic::libmagic)
